pyqtorch/core/batched_operation.py

```diff
@@ -330,3 +330,40 @@ def batchedRYY(
         state = RX(-torch.tensor(np.pi / 2), state, [q], N_qubits)
 
     return state
+    
```

Comment:
@dominikandreasseitz I guess batched `CRX`, `CRY` and `CRZ` are still missing, right? It should be fairly straightforward to implement them.

Reply:
they are implemented now